docs/sources/index.md

```diff
@@ -0,0 +1,50 @@
+# Triage
```

Comment:
Perhaps out of scope for this, but could/should the docs just link to the README here?

Reply:
The current functionality is a bit weird, but I think having the doc main page being just a link is a bad interface. There is a script that builds the API docs, maybe that script could also copy the README to this location?

Or else we make this index page have different content than the README.

Reply:
Sure. Copying the README might work well.

We could even convert between mark-up languages pretty easily –

    import pypandoc
    pypandoc.convert(…, 'md')

­– (assuming Python; we might be just using the pandoc executable).